02-express-tutorial/app.js

```diff
@@ -1 +1,63 @@
-console.log('Express Tutorial')
+const { products } = require("./data");
+const express = require('express');
+
+console.log('Express Tutorial');
+
+const app = express();
+
+app.use(express.static('./public'));
+
+
+app.get('/api/v1/test', (req, res) => {
+    res.json(products);
```

Comment:
I don't think its that important to change but just call our that the instructions said that the test route should return: `res.json({ message: "It worked!" });` and this does not.